internal/tfengine/tfengine.go

```diff
@@ -101,3 +102,41 @@ func dump(conf *Config, root, outputPath string) error {
 	}
 	return nil
 }
+
+// backendRE is a regex to capture GCS backend blocks in configs.
+// The 's' and 'U' flags allow capturing multi line backend blocks in a lazy manner.
+var backendRE = regexp.MustCompile(`(?sU)backend "gcs" {.*}`)
+
+// ConvertToLocalBackend converts all the Terraform backend blocks to "local".
+// path should be a path to the root of the configs generated by the Engine.
+func ConvertToLocalBackend(path string) error {
```

Comment:
Seems like this should go in a helper package. Maybe tfutil?

Reply:
I'm not sure. On the one hand it's specifically for the engine (to adjust its generated configs), on the other hand it would work on any .tf file. @xingao267 Any thoughts?

Reply:
Keeping it in the `tfengine` seems fine to me as it's pretty specific to the engine and currently not used by other tools.

Reply:
I don't have a strong opinion either way, just seeing tfengine.ConvertToLocalBackend seemed a little odd to me as it does not seem to be the engine's responsibility to be exporting this, nor is it actually used by the engine itself but rather only in the tests of the engine.